sdk/sdk/OBAApplication.h

```diff
@@ -0,0 +1,44 @@
+//
+//  sdk.h
+//  sdk
+//
+//  Created by Dima Belov on 4/25/15.
+//  Copyright (c) 2015 One Bus Away. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "OBAReferencesV2.h"
+#import "OBAModelDAO.h"
+#import "OBAModelService.h"
+#import "OBALocationManager.h"
+
+@interface OBAApplication : NSObject
+
+@property(nonatomic, strong, readonly) OBAReferencesV2 * references;
+@property(nonatomic, strong, readonly) OBAModelDAO * modelDao;
+@property(nonatomic, strong, readonly) OBAModelService * modelService;
+@property(nonatomic, strong, readonly) OBALocationManager * locationManager;
+
+/**
+ *  This block, if set, is called whenever refreshSettings also refreshes a region.
+ */
+@property (nonatomic, copy) dispatch_block_t regionRefreshed;
```

Comment:
I'd like to see the block removed and replaced with either a delegate method or a notification. Including a block on an asynchronously-executing method call is sensible and easy-to-understand to me, but it quickly becomes thorny when you're setting a single property on a single object, especially since any other object in the system can overwrite the block you originally passed in.